front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -244,6 +252,10 @@ AgentTablesQueryAction.init(
         fields: ["agentMessageId"],
         concurrently: true,
       },
+      {
+        fields: ["resultsFileId"],
```

Comment:
we need an index on this one ?

Reply:
I add it by default when I add a foreign key. If we need to delete the file for whatever reason it makes the SET NULL operation much cheaper.